### PR TITLE
MongoDB - update to version 5.0.1

### PR DESCRIPTION
**Pull request by EugeneKorolevich**

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -35,4 +35,22 @@ versions:
...
+  from: marketplace.gcr.io/google/debian9
+  packages:
+    gosu:
+      gpg: B42F6819007F00F88E364FD4036A9C25BF357DD4
+      version: '1.12'
+    mongodb:
+      gpg: F5679A222C647C87527C2F8CB00A0BD1E2C63C11
+      major: '5.0'
+      version: 5.0.1
+  repo: mongodb4
```
Comment (ovk6) [inline versions.yaml]: mongodb5

File: versions.yaml [hunk 2]
```diff
@@ -35,4 +35,22 @@ versions:
   - 4.4.5
   - '4.4'
   - '4'
```
Comment (ovk6) [inline versions.yaml:37]: latest tag should be there also, as it is latest for `mongodb4` repo

**Review by ovk6 — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 1 above)
Comment (EugeneKorolevich) [reply versions.yaml]: fixed

File: versions.yaml (hunk 2 above)
Comment (EugeneKorolevich) [reply versions.yaml:37]: fixed

**Review by ovk6 — APPROVED** [review]

**Pull request merged**